user_manager/api/v1/views.py

```diff
@@ -78,6 +79,37 @@ def authentication_classes(self):  # pragma: no cover
         ]
 
 
+class BulkCreateMixin(object):
+    """Allows bulk creation of a resource."""
+    def __init__(self):
+        self.bulk_operation = False
+        self.errors = []
+
+    def get_serializer(self, *args, **kwargs):
+        if isinstance(kwargs.get('data', {}), list):
+            self.bulk_operation = True
+            kwargs['many'] = True
+
+        return super(BulkCreateMixin, self).get_serializer(*args, **kwargs)
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        reports = self.perform_create(serializer)
+        headers = self.get_success_headers(serializer.data)
+
+        status_code = status.HTTP_201_CREATED
+        serialized_reports = serializer.to_representation(reports)
+
+        data = {'results': serialized_reports} if self.bulk_operation else serialized_reports
+
+        if self.errors:
+            status_code = status.HTTP_404_NOT_FOUND
```

Comment:
I don't think this is the right status, please remove this, a 201 or 202 is fine.

Reply:
Fixed to `201`